packages/cli/test/acceptance/app-run.acceptance.js

```diff
@@ -68,10 +68,12 @@ describe('app-generator (SLOW)', function() {
   });
 });
 
-function lernaBootstrap(scope) {
-  const cmd = new lerna.BootstrapCommand('', {
+async function lernaBootstrap(scope) {
+  const cmd = bootstrapCommandFactory({
+    _: [],
+    ci: false,
     scope: scope,
     loglevel: 'silent',
   });
-  return cmd.run();
+  await cmd;
```

Comment:
How does just awaiting a command trigger it? 🤔

Reply:
Please note that `cmd` is a promise-like object when runs the command itself.